boards/PDM/Inc/main.h

```diff
@@ -29,7 +29,7 @@ extern "C"
 #endif
 
 /* Includes ------------------------------------------------------------------*/
-#include "stm32f3xx_hal.h"
+#include "stm32f4xx_hal.h"
```

Comment:
```suggestion
#include <Io_Hal.h>
```

Reply:
this is autogenerated so not gonna edit here, but otherwise good call